packages/greencheck/tests/SitecheckCachingTest.php

```diff
@@ -29,14 +31,19 @@ public function setUp(): void
         // Setup the cache
         $cache = new Sitecheck\Cache($config);
         $cache->setCache('default');
+        $redisCache = $cache->getCache();
+        $redis = $redisCache->getRedis();
+
+        $logger = new Sitecheck\Logger($entityManager, $redis);
```

Comment:
if we don't really care about the logger in our tests, maybe we should just mock it instead.